public/css/restaurant-home.css

```diff
@@ -0,0 +1,150 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#F2F2F2;
+    --food-slider-bg:#171A29;
+    --text-col-white:#FFFFFF;
+    --text-col-gray:#9EA0A6;
+    --radio-color:#4f46e5;
+}
+.navbar{
+    padding: 1rem 1rem 0;
+}
+.nav-zindex{
+    z-index: 999;
+}
+
+.discount-container{
+    height: 35vh;
+    background: var(--food-slider-bg);
+    position: relative;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+}
+
+
+
+.hotel-img{
+    height: 15.625rem;
+    width: 20.625rem;
+    border-radius: 7px;
+    min-width: 14.375rem;
+    min-height: 14.375rem;
+    max-height: 14.375rem;
+    max-width: 21.875rem;
+    flex: 1;
+    object-fit: cover;
+}
+
+
+
+
+.hotelname{
+    font-size: 2.6rem;
+    color: var(--text-col-white);
+    font-weight: 800;
+}
+
+.hoteldesc{
+    font-size: 1.2rem;
+    color: var(--text-col-gray);
+    font-weight: 400;
+    
+}
+
+
+
+.divider{
+    border-right: 2px solid var(--text-col-gray);
+}
+.txt-sm{
+    font-size: 1rem;
+    font-weight: 500;
+    color: var(--text-col-gray);
+}
+
+.txt-md{
+    font-size: 1.4rem;
+    font-weight: 700;
+    color: var(--text-col-white);
+}
+
+.hotel-lists{
+    width: 95%;
+    margin: auto;
+    display: flex;
+}
+
+.filters{
+    width: 25%;
+    height: 75vh; 
+    min-height: 12.5rem;
+    overflow: auto;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 15%;
+}
+
+.menu-section{
+    display: flex;
+    justify-content: space-between;
+    gap: 10px;
+    margin-top: 1;
+}
+.search-menu{
+    width: 100%;
+}
+.menu-lists{
+    flex: 7;
+    gap:1rem;
+}
+
+.radio{
+    accent-color: var(--radio-color);
+}
+
+.qg-flex-1{
```

Comment:
the naming is throwing me off a bit, can't seem to figure out what qg stands from

Reply:
> the naming is throwing me off a bit, can't seem to figure out what qg stands from

@amantulsyan35 .**qg** it's just a prefix value, I have added cause of not to collide with the Component library.